src/Utils.ps1

```diff
@@ -282,6 +282,27 @@ function Get-PathStringComparison {
     }
 }
 
+function Get-PromptPath {
```

Comment:
It's tempting to add parameters to this function Iike `-AbbreviateHomeDir` and `-PathTruncateLimit` which would eliminate the need for the equivalent $GitPromptSettings but the "discoverability" of this functionality would be very poor.  Probably best to leave those in $GitPromptSettings.

Reply:
That's exactly how I started, but moved away from it in favor of keeping the default `DefaultPromptPath` text simple (`'$(Get-PromptPath)').